ci/scripts/utils/ci_utils.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/env bash
+
+function cancel_slurm_jobs() {
+
+  local substring=$1
+  local job_ids
+  job_ids=$(squeue -u "${USER}" -h -o "%i")
+
+  for job_id in ${job_ids}; do
+    job_name=$(sacct -j "${job_id}" --format=JobName%100 | head -3 | tail -1 | sed -r 's/\s+//g') || true
+    if [[ "${job_name}" == *"${substring}"* ]]; then
```

Comment:
```suggestion
    if [[ "${job_name}" =~ "${substring}" ]]; then
```

Reply:
didn't know about this construct, it will still pick up substrings with this operator?

Reply:
Yes, it is a regex matcher.

Reply:
Did this change get undone at some point? It is showing as `==` again.

Reply:
Updated suggestion, since shellcheck was complaining:
```suggestion
    if [[ "${job_name}" =~ ${substring} ]]; then
```